protobuf_deps.bzl

```diff
@@ -13,6 +13,8 @@ PROTOBUF_MAVEN_ARTIFACTS = [
     "com.google.truth:truth:1.1.2",
     "junit:junit:4.13.2",
     "org.mockito:mockito-core:4.3.1",
+    "biz.aQute.bnd:biz.aQute.bndlib:6.4.0",
+    "info.picocli:picocli:4.6.3",
```

Comment:
Where are these coming from?

Reply:
Maven?
To make the WrapOsgi tool work
bndlib is the library that actually computes and writes the Manifest file.